api/scpca_portal/metadata_file.py

```diff
@@ -87,7 +88,26 @@ def transform_keys(data_dict: Dict, key_transforms: List[Tuple]):
     return data_dict
 
 
-def write_metadata_dicts(list_of_dicts: List[Dict], output_file_path: str, **kwargs) -> None:
+class MetadataFilenames:
+    SINGLE_CELL_METADATA_FILE_NAME = "single_cell_metadata.tsv"
+    SPATIAL_METADATA_FILE_NAME = "spatial_metadata.tsv"
+    METADATA_ONLY_FILE_NAME = "metadata.tsv"
```

Comment:
I wasn't sure if these constants should be higher up in the file (as we do conventionally with constants), or if they should be close to the place where they're called. I decided to put them here for enhanced clarity during the PR review (so that it would be clear what `get_metadata_file_name` is doing).

Reply:
This is fine. You don't really care what the value is, just that it is consistent across the application.